sdk/storage/src/blobs/block_blob_client.cpp

```diff
@@ -75,6 +77,64 @@ namespace Azure { namespace Storage { namespace Blobs {
         options.Context, *m_pipeline, m_blobUrl.ToString(), content, protocolLayerOptions);
   }
 
+  BlobContentInfo BlockBlobClient::UploadFromBuffer(
+      const uint8_t* buffer,
+      std::size_t bufferSize,
+      const UploadBlobOptions& options) const
+  {
+    constexpr int64_t c_defaultBlockSize = 8 * 1024 * 1024;
```

Comment:
`c_defaultBlockSize ` should be a non-nullable optional parameter with a default value of 8MB instead.

Reply:
In `options` we already have a nullable `ChunkSize`, which will override this default value if provided.

What's the point of having them both in `options`?

Reply:
Then I don't think it's neccessary to have it here. We should have `ChunkSize` default to 8MB.